src/main/kotlin/no/nav/syfo/services/UpdateInfotrygdService.kt

```diff
@@ -438,8 +441,12 @@ suspend fun sendInfotrygdOppdateringAndValidationResult(
                 applicationState.ready = false
             }
 
+            val dropInfotrygdUpdate = dropInfotrygdUpdate(receivedSykmelding, validationResult)
```

Comment:
Kan vi gi denne et mer forståelig norsk navn? `skalIkkeOppdatereInfotrygd`, for eksempel? :)

Reply:
Samme gjelder metoden under :)

Reply:
Jepp